cli/cmd/turbo/main.go

```diff
@@ -4,27 +4,13 @@ import "C"
 import (
 	"encoding/json"
 	"fmt"
-	"os"
-	"unsafe"
 
 	"github.com/vercel/turbo/cli/internal/cmd"
 	"github.com/vercel/turbo/cli/internal/turbostate"
 )
 
 func main() {
-	os.Exit(cmd.RunWithArgs(os.Args[1:], turboVersion))
-}
-
-//export nativeRunWithArgs
-func nativeRunWithArgs(argc C.int, argv **C.char) C.uint {
-	arglen := int(argc)
-	args := make([]string, arglen)
-	for i, arg := range unsafe.Slice(argv, arglen) {
-		args[i] = C.GoString(arg)
-	}
-
-	exitCode := cmd.RunWithArgs(args, turboVersion)
-	return C.uint(exitCode)
+	fmt.Printf("ERROR: Go binary cannot be used on its own. Please build as c-archive and use with Rust crate")
```

Comment:
We should have a non-zero exit code in case this gets called by a script that doesn't forward stdout